weave/flow/__init__.py

```diff
@@ -0,0 +1,15 @@
+from weave.flow.agent import Agent, AgentState
+from weave.flow.dataset import Dataset
+from weave.flow.eval import Evaluation, Scorer
+from weave.flow.model import Model
+from weave.flow.obj import Object
+
+__docspec__ = [
```

Comment:
docspec is only needed if we want this particular file to be doced. Specifically, we don't want to doc-spec something twice. This can be removed

Reply:
Just to be clear: `__docspec__` is only used by our doc generator. We should expect each symbol to appear in only one `__docspec__` list. Since these are already in the top-level init, no need to have them here